include/dxc/HLSL/HLModule.h

```diff
@@ -211,6 +211,9 @@ class HLModule {
   static bool HasPreciseAttributeWithMetadata(llvm::Instruction *I);
   static void MarkPreciseAttributeWithMetadata(llvm::Instruction *I);
   static void ClearPreciseAttributeWithMetadata(llvm::Instruction *I);
+  template<class BuilderTy>
+  static void MarkPreciseAttributeOnValWithFunctionCall(llvm::Value *V,
+							BuilderTy &Builder, llvm::Module &M);
```

Comment:
Tabs here.  Make sure your settings are to insert 2-space indents.

Reply:
done!